src/components/pages/pseudochoice.tsx

```diff
@@ -4,6 +4,46 @@ import React from 'react';
 //   onCorrect: () => void;
 // }
 
+function MoveTurnComponent(input){
```

Comment:
No need to have the word component, let's just call it MoveTurn or maybe MoveTurnInstruction